app/screens/channel/channel_post_list/index.js

```diff
@@ -26,7 +27,7 @@ function mapStateToProps(state) {
         deviceHeight: state.device.dimension.deviceHeight,
         postIds: getPostIdsInCurrentChannel(state),
         postVisibility: state.views.channel.postVisibility[channelId],
-        lastViewedAt: getMyCurrentChannelMembership(state).last_viewed_at,
+        lastViewedAt: currentChannelMember && currentChannelMember.last_viewed_at,
```

Comment:
Since this is RN, we can actually change this to use `?.` if you want to simplify it
```suggestion
        lastViewedAt: currentChannelMember?.last_viewed_at,
```

Reply:
Good point

Reply:
This doesn't work because the eslint, so I change it again to my previous approach.